DigitalLearningSolutions.Data/Models/DelegateGroups/Group.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace DigitalLearningSolutions.Data.Models.DelegateGroups
+{
+    public class Group
+    {
+        public int GroupId { get; set; }
+
+        public string GroupLabel { get; set; }
+
+        public string? GroupDescription { get; set; }
+
+        public int DelegateCount { get; set; }
+
+        public int CoursesCount { get; set; }
+
+        public string AddedByFirstName { get; set; }
+
+        public string AddedByLastName { get; set; }
+
+        public int LinkedToField { get; set; }
+
+        public string LinkedToFieldName { get; set; }
+
+        public bool AddNewRegistrants { get; set; }
+
+        public bool SyncFieldChanges { get; set; }

```

Comment:
So, obviously this was the db creator's choice, but I don't know what `AddNewRegistrants=true` means - does it mean that new registrants should be added to the group? Or have been added? I have similar issues with `SyncFieldChanges=true`.

I would be inclined to give these boolean properties more helpful names and use a mapper (take inspiration from `CurrentCourseMap`) to map them to the db properties.